client/src/test/java/com/aws/greengrass/cli/commands/LogsTest.java

```diff
@@ -216,13 +217,15 @@ void testGetSyslogHappyCase() throws InterruptedException {
     void testGetSyslogFollowHappyCase() throws InterruptedException {
         fileWriter.println(syslogEntry1);
         Thread thread = new Thread(() -> runCommandLine("logs", "get", "--log-file", logFile.toString(),
-                "--time-window", YEAR + "-08-30,+365d", "--follow", "--syslog", "--no-color"));
+                "--time-window", YEAR + "-01-01,+5s", "--follow", "--syslog", "--no-color"));
```

Comment:
The reason for this test to fail on Windows was because:
1. The time window was set to end *one year* in the future.
2. A thread pool is used to execute a task that reads the log file. The task has a loop which won't exit until the time window has ended.
3. The task is holding on to a log file in the temp directory.
4. The test tried to interrupt the thread. But the thread is not the same as the thread pool.
4. When the test ends, JUnit tries to remove the temp directory. But because a task (running in a thread pool) is still holding on the log file, therefore JUnit cannot remove the temp directory and throws an exception. 
5. Because of the exception, the test was marked as failed.

The fix is to set the time window to just 5 second (",+5s") in the future. I also have to change what are being sent to the log so that those logs would fit into the new smaller time window.

Reply:
I think I'd prefer to properly kill the task which is running so that it releases the locks

Reply:
The way `LogsTest` is written by creating a new `CLI` `CommandLine` ([here](https://github.com/aws-greengrass/aws-greengrass-cli/blob/main/client/src/test/java/com/aws/greengrass/cli/commands/LogsTest.java#L255)) instead of creating `Logs`, I can't get a hold of the `Future` returned from the `ExecutorService` ([here](https://github.com/aws-greengrass/aws-greengrass-cli/blob/main/client/src/main/java/com/aws/greengrass/cli/util/logs/impl/AggregationImpl.java#L97)).